### ISSUE TITLE
repl: yaegi panics with go built from source

### ISSUE THREAD

**Issue by sbinet** [containous/yaegi#309]
running `yaegi` with a Go environment built from `master` (or from any branch), panics like so:

```
$> yaegi
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/containous/yaegi/interp.goNumVersion(0x2)
	/home/binet/dev/go/gocode/src/github.com/containous/yaegi/interp/build.go:91 +0xa5
github.com/containous/yaegi/interp.init()
	/home/binet/dev/go/gocode/src/github.com/containous/yaegi/interp/build.go:60 +0x26
```

that's because:
```
$> go version
go version devel +60f14fddfe Wed Jun 26 16:35:14 2019 +0000 linux/amd64
```

and `build.go:91` looks like:
```go
// goNumVersion returns the go minor version number
func goNumVersion() int {
        v := strings.Split(runtime.Version(), ".")
        n, _ := strconv.Atoi(v[1])
        return n
}
```

**Comment by ldez** [comment]
Related to #304 

Fixed by #306

**Comment by ldez** [comment]
The REPL need the [stdlib mapping](https://github.com/containous/yaegi/tree/master/stdlib).

We provide only go1.11 and go1.12, as soon as the first stable 1.13 will be available we will add the support.

For the `devel` version it's possible to build a stable version of Yaegi with it, in this case you need to generate your stdlib mapping (with the `goexports` comand) and compile your own version of yaegi.
